scripts/run_design.py

```diff
@@ -90,8 +92,15 @@ def main():
     for arg_name, enum in flow_args_map.items():
         flow_args[enum] = getattr(args, arg_name)
 
-    run_design(Path(args.design_path), args.flow, args.error_flow, flow_args=flow_args)
+    # Convert the design path to an absolute path
+    design_path = Path(os.path.expanduser(args.design_path))
```

Comment:
What is the call to `expanduser` meant to do?  This function expands the `~` in a path, but it would have already been done by the shell prior to this code running, so I'm not sure what it's accomplishing here?

Reply:
reverted